data/FilterList.json

```diff
@@ -42,7 +41,9 @@
     "publishedDate": "2016-11-19T20:26:13",
     "syntaxId": 1,
     "updatedDate": "2016-12-29T17:46:08",
-    "viewUrl": "https://raw.githubusercontent.com/michaeltrimm/hosts-blocking/master/_hosts.txt"
+    "viewUrl": "https://raw.githubusercontent.com/michaeltrimm/hosts-blocking/master/_hosts.txt",
+    "viewUrlMirror1": "https://gitcdn.xyz/repo/michaeltrimm/hosts-blocking/master/_hosts.txt",
```

Comment:
Might be automated in the future if we implement #598 , but this is great for now.